Detections/Syslog/ssh_NewlyInternetExposed.yaml

```diff
@@ -0,0 +1,57 @@
+﻿id: 4915c713-ab38-432e-800b-8e2d46933de6
+name: ssh newly internet-exposed endpoints
+description: endpoints with a history of sign-ins only from private IP addresses are accessed from a public IP address.
```

Comment:
Capitalization on Endpoints, as this shows in the UX

Reply:
done